korman/exporter/physics.py

```diff
@@ -165,33 +165,36 @@ def generate_physical(self, bo, so, **kwargs):
                 physical.friction = mod.friction
                 physical.restitution = mod.restitution
 
-                if mod.dynamic:
-                    if ver <= pvPots:
-                        physical.collideGroup = (1 << plSimDefs.kGroupDynamic) | \
-                                                (1 << plSimDefs.kGroupStatic)
-                    physical.memberGroup = plSimDefs.kGroupDynamic
-                    physical.mass = mod.mass
-                    _set_phys_prop(plSimulationInterface.kStartInactive, simIface, physical,
-                                   value=mod.start_asleep)
-                elif not mod.avatar_blocker:
-                    physical.memberGroup = plSimDefs.kGroupLOSOnly
+                if not mod.dynamic_blocker:
+                    if mod.dynamic:
```

Comment:
This combination indicates to me that kickables cannot collide with each other.

Reply:
> This combination indicates to me that kickables cannot collide with each other.

I changed that section of the code back to how it was while adding and modifying a new line that'll hopefully do the trick.